scripts/dev/setup_kind_cluster.sh

```diff
@@ -5,24 +5,27 @@ function usage() {
   echo "Deploy local registry and create kind cluster configured to use this registry. Local Docker registry is deployed at localhost:5000.
 
 Usage:
-  setup_kind_cluster.sh [-n <cluster_name>]
+  setup_kind_cluster.sh [-n <cluster_name>] [-f]
```

Comment:
can we name it -r, like "recreate"?
"force" for me is something that we're doing despite some circumstances, e.g. errors.